Add Defender for Endpoint File Indicator to file statistics command (#37022)
demisto/content: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

| --- | --- | --- |
| MicrosoftATP.FileStatistics.Sha1 | String | The file SHA1 hash. | 
| MicrosoftATP.FileStatistics.Statistics.OrgPrevalence | String | The prevalence of the file in the organization. | 

> | MicrosoftATP.FileStatistics.Statistics.OrganizationPrevalence | Number | The prevalence of the file in the organization. | 

Reviewer: same here

> | MicrosoftATP.FileStatistics.Statistics.OrgPrevalence | String | The prevalence of the file in the organization. | 
> | MicrosoftATP.FileStatistics.Statistics.OrganizationPrevalence | Number | The prevalence of the file in the organization. | 

Reviewer 2: ```suggestion
| MicrosoftATP.FileStatistics.Statistics.OrgPrevalence | String | The number of times the file is detected in the organization. | 
| MicrosoftATP.FileStatistics.Statistics.OrganizationPrevalence | Number | The number of times the file is detected in the organization. | 
```

| MicrosoftATP.FileStatistics.Statistics.OrgFirstSeen | Date | The first date and time the file was seen in the organization. | 
| MicrosoftATP.FileStatistics.Statistics.OrgLastSeen | Date | The last date and time the file was seen in the organization. | 
| MicrosoftATP.FileStatistics.Statistics.GlobalPrevalence | String | The global prevalence of the file. | 

> | MicrosoftATP.FileStatistics.Statistics.GloballyPrevalence | Number | The global prevalence of the file. | 

Reviewer 2: ```suggestion
| MicrosoftATP.FileStatistics.Statistics.GloballyPrevalence | Number | The number of times the file is detected globally. | 
```

| MicrosoftATP.FileStatistics.Statistics.GlobalFirstObserved | Date | The first global observation date and time of the file. | 
| MicrosoftATP.FileStatistics.Statistics.GlobalLastObserved | Date | The last global observation date and time of the file. | 
| MicrosoftATP.FileStatistics.Statistics.TopFileNames | String | The top names of the file. | 
| File.SHA1 | String | The SHA1 hash of the file. |

> | File.OrganizationPrevalence | Number | How frequently the indicator is detected within a particular organization. |
> | File.GlobalPrevalence | Number | How frequently the indicator is detected across all organizations. |

Reviewer 2: ```suggestion
| File.OrganizationPrevalence | Number | The number of times the indicator is detected within a particular organization. |
| File.GlobalPrevalence | Number | The number of times the indicator is detected across all organizations. |
```

| 2019-04-03T04:10:18.1001071Z | 2020-03-23T09:24:54.169574Z | 1355899 | 0 | lsass.exe |

##### File Indicator Example

> The basic indicator fields from the command are shown below without auto extraction and enrichment.

Reviewer: let's discuss this

Author: Sentence removed (as we discussed in person)
